src/webapp/pages/form-page/disease-outbreak-event/utils/applyRulesInFormState.ts

```diff
@@ -0,0 +1,89 @@
+import { Rule } from "../../../../../domain/entities/Rule";
```

Comment:
file name should start with capital letter unless it is a hook right?

Reply:
Same as the other PR, let's discuss it at the technical meeting and agree on one to use it